core/include/earcut.hpp

```diff
@@ -1 +1 @@
-Subproject commit 1384c97a79c322e1aad37fd317bf87be5dbb24fa
+Subproject commit 195b2fdae9a354c0f86592f320df4988c896f822
```

Comment:
This update looks fine to me. Have you checked whether these issues are fixed in the original mapbox repo?